[2023-03-29 14:24:48,189] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_2222.date_update_mrr 2023-03-28T00:00:00+00:00 [queued]>
[2023-03-29 14:24:48,205] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_2222.date_update_mrr 2023-03-28T00:00:00+00:00 [queued]>
[2023-03-29 14:24:48,207] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 14:24:48,210] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-03-29 14:24:48,212] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 14:24:48,236] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): date_update_mrr> on 2023-03-28T00:00:00+00:00
[2023-03-29 14:24:48,242] {standard_task_runner.py:52} INFO - Started process 20909 to run task
[2023-03-29 14:24:48,254] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_2222', 'date_update_mrr', '2023-03-28T00:00:00+00:00', '--job-id', '148', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmptusili6c', '--error-file', '/tmp/tmpy6e8izx7']
[2023-03-29 14:24:48,256] {standard_task_runner.py:77} INFO - Job 148: Subtask date_update_mrr
[2023-03-29 14:24:48,345] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_2222.date_update_mrr 2023-03-28T00:00:00+00:00 [running]> on host 0bc3da97cebb
[2023-03-29 14:24:48,434] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_2222
AIRFLOW_CTX_TASK_ID=date_update_mrr
AIRFLOW_CTX_EXECUTION_DATE=2023-03-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-28T00:00:00+00:00
[2023-03-29 14:24:48,489] {python.py:151} INFO - Done. Returned value was: None
[2023-03-29 14:24:48,521] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=postgres_2222, task_id=date_update_mrr, execution_date=20230328T000000, start_date=20230329T142448, end_date=20230329T142448
[2023-03-29 14:24:48,553] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-03-29 14:24:48,587] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
